### DISCUSSION TITLE
Sponsor

### DISCUSSION THREAD

**Discussion by cpatti97100** [localsend/localsend#745] [Discussion (English, International)]
Hi could you please provide alternative ways to sponsor?

Thanks

**Comment by Tienisto** [comment]
Hello, what methods do you have in mind? :)

**Reply by cpatti97100** [reply]
not sure but I noticed PayPal is not allowed anymore here on GitHub for a 1 time donation :(

**Reply by mertssmnoglu** [reply]
There are multiple sponsorship methods.
Maybe you can check buymeacoffee, kofi, patreon, or opencollective.

**Reply by cpatti97100** [reply]
oh I see, do you have a link to a page where those are mentioned?

**Reply by mertssmnoglu** [reply]
All available funding methods are listed in `.github/FUNDING.yml`. [The developer and maintainer](https://github.com/Tienisto) does not appear to be using these services.
@cpatti97100 Can you choose a service that is easy for you and write about it? This will makes the process easier.
